packages/core/src/actors.ts

```diff
@@ -225,7 +225,7 @@ export function fromPromise<T>(
     getInitialState: () => {
       return {
         canceled: false,
-        status: 'pending',
+        status: 'active',
```

Comment:
this just unified the name of the status with the name used by observable actors